api/s2n.h

```diff
@@ -66,6 +66,15 @@ typedef enum {
     S2N_EXTENSION_CERTIFICATE_TRANSPARENCY = 18,
 } s2n_tls_extension_type;
 
+typedef enum {
+    S2N_TLS_MAX_FRAG_LEN_EXT_NONE,     /* don't use the extension */
```

Comment:
This was to explicitly show that valid mfl enum value starts at 1 not 0. 
Also more clear when initializing mfl_code values during new config creation.